<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Forgot_Password</class>
 <widget class="QDialog" name="Forgot_Password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Forgot_Password</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>132</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="edit_username"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="edit_code"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btn_sendCode">
      <property name="text">
       <string>发送验证码</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>验证码：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>电子邮箱</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edit_email"/>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btn_commit">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edit_username</tabstop>
  <tabstop>edit_email</tabstop>
  <tabstop>btn_sendCode</tabstop>
  <tabstop>edit_code</tabstop>
  <tabstop>btn_commit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
